tests/e2e/support/objects/app-admin-settings/groups/index.ts

```diff
@@ -16,8 +16,7 @@ export class Groups {
     this.#page = page
   }
   getUUID({ key }: { key: string }) {
-    const { uuid } = this.#usersEnvironment.getGroup({ key })
-    return uuid
+    return this.#usersEnvironment.getGroup({ key }).uuid
```

Comment:
can we add a line break between 17 and 18? Also a return type of the function?

Reply:
yeah, we can but probably in next PRs